sql/src/main/java/io/crate/metadata/information/InformationPartitionsTableInfo.java

```diff
@@ -105,35 +107,57 @@ private static ColumnRegistrar createColumnRegistrar() {
-            .register(InformationTablesTableInfo.Columns.TABLE_VERSION_UPGRADED, DataTypes.STRING)
             .register(InformationTablesTableInfo.Columns.CLOSED, DataTypes.BOOLEAN)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS, DataTypes.OBJECT)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_BLOCKS, DataTypes.OBJECT)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_BLOCKS_READ_ONLY, DataTypes.BOOLEAN)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_BLOCKS_READ, DataTypes.BOOLEAN)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_BLOCKS_WRITE, DataTypes.BOOLEAN)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_BLOCKS_METADATA, DataTypes.BOOLEAN)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_TRANSLOG, DataTypes.OBJECT)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_TRANSLOG_FLUSH_THRESHOLD_SIZE, DataTypes.LONG)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_TRANSLOG_SYNC_INTERVAL, DataTypes.LONG)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_ROUTING, DataTypes.OBJECT)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_ROUTING_ALLOCATION, DataTypes.OBJECT)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_ROUTING_ALLOCATION_ENABLE, DataTypes.STRING)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_ROUTING_ALLOCATION_TOTAL_SHARDS_PER_NODE, DataTypes.INTEGER)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_WARMER, DataTypes.OBJECT)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_WARMER_ENABLED, DataTypes.BOOLEAN)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_UNASSIGNED, DataTypes.OBJECT)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_UNASSIGNED_NODE_LEFT, DataTypes.OBJECT)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_UNASSIGNED_NODE_LEFT_DELAYED_TIMEOUT, DataTypes.LONG)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_MAPPING, DataTypes.OBJECT)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_MAPPING_TOTAL_FIELDS, DataTypes.OBJECT)
-            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS_MAPPING_TOTAL_FIELDS_LIMIT, DataTypes.INTEGER);
+            .register(InformationTablesTableInfo.Columns.TABLE_VERSION, ObjectType.builder()
+                .setInnerType(Version.Property.CREATED.toString(), DataTypes.STRING)
+                .setInnerType(Version.Property.UPGRADED.toString(), DataTypes.STRING)
+                .build())
+            .register(InformationTablesTableInfo.Columns.TABLE_SETTINGS, ObjectType.builder()
+                //.setInnerType("refresh_interval", DataTypes.LONG)
```

Comment:
Can this be removed?

Reply:
Yes, I'll create a followup task in order to match partition settings with normal table settings.